src/lib/routeLoaders.ts

```diff
@@ -122,6 +123,10 @@ export const fileLoader: LoaderFunction = async (
       })
       code = normalizeLineEndings(code)
 
+      // If persistCode in localStorage is present, it'll persist that code
+      // through *anything*. INTENDED FOR TESTS.
+      code = codeManager.localStoragePersistCode() || code
```

Comment:
```suggestion
      if (window.electron.process.env.IS_PLAYWRIGHT) {
          code = codeManager.localStoragePersistCode() || code
      }
```

Reply:
I would feel much better about this if it were guarded. I don't know if we do it anymore but we used to persist the user's code on desktop to both the file system and localStorage and use the latter as a fallback. If that's still the case then under normal use this runs the risk of overwriting the user's code when they navigate with their last file's code.